CHANGELOG.md

```diff
@@ -21,6 +21,8 @@ For older changes see the [archived Singularity change log](https://github.com/a
   and resulted in a failure when pulling OCI containers.
 - Add a clear error message if someone tries to use privileged network options
   while not using setuid mode.
+- Allow multi-arch oci-archive files, that have nested index with the manifest.
+  This is the default format (both Docker and OCI), when using `nerdctl save`.
```

Comment:
Please remove the comma in both sentences, and `a` before `nested`, and `for` before `both`.